public/src/components/channelManagement/epicTests/epicVariantPreview.tsx

```diff
@@ -125,4 +127,35 @@ const EpicVariantPreview: React.FC<EpicVariantPreviewProps> = ({
   return <div className={classes.container}>{Epic && <Epic {...props} />}</div>;
 };
 
+const EpicVariantPreviewStorybook = ({ variant }: { variant: EpicVariant }) => {
+  const classes = useStyles();
+
+  const tickerSettingsWithData = useTickerData(variant.tickerSettings);
+  const props = buildProps(variant, tickerSettingsWithData);
+  const compressedProps = lzstring.compressToEncodedURIComponent(JSON.stringify(props));
+
+  const storyName = 'components-marketing-contributionsliveblogepic--default';
+  const dcrStorybookUrl = 'https://5dfcbf3012392c0020e7140b-borimwnbdl.chromatic.com';
```

Comment:
it looks random but this is static!

Reply:
Currently this would only work against the PROD DCR storybook.
@jamesgorrie do you know if there's a CODE storybook?

Reply:
> it looks random but this is static

Good to know! I always wondered if these URLs ever changed.

Reply:
There isn't a `CODE` environment as Chromatic and storybook deploy to feature branches rather than environments.

Do we need a CODE environment to support a specific workflow?